pkg/codegen/testing/test/testdata/mini-azurenative/java/src/main/java/com/pulumi/azurenative/alertsmanagement/outputs/GetActionRuleByNameResult.java

```diff
@@ -51,7 +51,7 @@ private GetActionRuleByNameResult() {}
      * @return Azure resource Id
      * 
      */
-    public String id() {
+    public String id_() {
```

Comment:
Uhm this seems like a breaking change that's really undesirable? Can this work be accomplished without renaming id to id_ in providers in case some examples/ code out there already is linking to id()?

Reply:
GetActionRuleByNameResult is not a resource. It seems like isReservedResourceMethod is perhaps mis-applied.

Reply:
> GetActionRuleByNameResult is not a resource. It seems like isReservedResourceMethod is perhaps mis-applied.

or miss-named, I'm not sure

Reply:
rebased on fix for this